bin/server_stop.sh

```diff
@@ -23,6 +23,7 @@ if [ ! -f "$pidFilePath" ] || ! kill -0 $(cat "$pidFilePath"); then
    echo 'Job server not running'
 else
   echo 'Stopping job server...'
-  kill -15 $(cat "$pidFilePath") && rm -f "$pidFilePath"
+  PID=$(cat "$pidFilePath")
+  kill -15 -- -$(ps -o pgid= $PID | grep -o [0-9]*) && rm -f "$pidFilePath"
```

Comment:
Unfortunately this didn't worked for me. Digging deeper I found that `grep -o` [is broken in some versions](http://stackoverflow.com/questions/1736439/why-does-this-grep-o-fail-and-how-should-i-work-around-it) and there is more versatile solution this:
```
egrep -o [0-9]+
```